src/translations/en.json

```diff
@@ -4755,6 +4755,8 @@
               "state_equal": "State is equal to",
               "state_not_equal": "State is not equal to",
               "current_state": "current",
+              "min_width": "Min width",
+              "max_width": "Max width",
```

Comment:
unused atm?

Reply:
No, I'm working on the UI editor for the responsive condition. I will remove unused keys.